db/seeders/20220918075630-seed-data.js

```diff
@@ -1,24 +1,57 @@
-'use strict';
-
 module.exports = {
-  async up (queryInterface, Sequelize) {
-    /**
-     * Add seed commands here.
-     *
-     * Example:
-     * await queryInterface.bulkInsert('People', [{
-     *   name: 'John Doe',
-     *   isBetaMember: false
-     * }], {});
-    */
+  up: async (queryInterface) => {
+    // Define user data
+    const userData = [
+      {
+        first_name: 'John',
+        last_name: 'Doe',
+        email: 'john@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
+        created_at: new Date(),
+        updated_at: new Date(),
+      },
+      {
+        first_name: 'Mary',
+        last_name: 'Jane',
+        email: 'mary@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
+        created_at: new Date(),
+        updated_at: new Date(),
+      },
+      {
+        first_name: 'Peter',
+        last_name: 'Griffin',
+        email: 'peter@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
+        created_at: new Date(),
+        updated_at: new Date(),
+      },
+      {
+        first_name: 'Ruth',
+        last_name: 'Boaz',
+        email: 'ruth@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
+        created_at: new Date(),
+        updated_at: new Date(),
+      },
+      {
+        first_name: 'Simon',
+        last_name: 'Says',
+        email: 'simon@gmail.com',
+        password: 'cec96aa1b911244e1657b54aadb408d60dcc742b7b3eede48e2fd7708589b048c6f7ac3fa6667e002f9ec80ec4d8d43fe24982a87ffe7f9f72be33fae6e3590c',
+        created_at: new Date(),
+        updated_at: new Date(),
+      },
+    ];
+
+    // Bulk insert users, returning=true,
+    // and destructure the returned results array, for use in other tables
+    const [johnUser, maryUser, peterUser, ruthUser, simonUser] = await queryInterface.bulkInsert('users', userData, {
```

Comment:
We don't really do anything with the output right. 

we can do
```suggestion
    await queryInterface.bulkInsert('users', userData)
```

Reply:
Was preparing for the eventual use of the output hahaha.